ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -758,7 +758,7 @@ Rectangle {
             }
         }
 
-        let textBeforeCursor = StatusQUtils.Emoji.deparseFromParse(plainText.substr(0, i));
+        let textBeforeCursor = StatusQUtils.Emoji.deparse(plainText.substr(0, i));
```

Comment:
How is this change related?

Same below

Reply:
This is coming with `Emoji.qml` changes.
I had to fix the paths there. And made some rexexp refactoring there. After that `deparseFromParse` was dropped, because afterall it was the same as `deparse`. 🙂